### PR TITLE
docs: Added SystemJS maps for CDK

### PR DESCRIPTION
**Pull request by shaunak1111**
adds necessary SystemJS maps for CDK individual packages to the docs

### CODE REVIEW

**Comment by shaunak1111** [comment]
This closes #7175 issue

File: guides/getting-started.md [hunk 1]
```diff
@@ -149,13 +149,22 @@ Here is a example configuration where `@angular/material`, `@angular/cdk/platfor
...
+    '@angular/cdk/coercion':'npm:/@angular/cdk/bundles/cdk-coercion.umd.js',
+    '@angular/cdk/collections':'npm:/@angular/cdk/bundles/cdk-collections.umd.js',
+    '@angular/cdk/keycodes':'npm:/@angular/cdk/bundles/cdk-keycodes.umd.js',
+    '@angular/cdk/observers':'npm:/@angular/cdk/bundles/cdk-observers.umd.js',
+    '@angular/cdk/overlay':'npm:/@angular/cdk/bundles/cdk-overlay.umd.js', 
+    '@angular/cdk/platform': 'npm:@angular/cdk/bundles/cdk-platform.umd.js',
+    '@angular/cdk/portal':'npm:/@angular/cdk/bundles/cdk-portal.umd.js',
+    '@angular/cdk/rxjs':'npm:/@angular/cdk/bundles/cdk-rxjs.umd.js',
+    '@angular/cdk/scrolling':'npm:/@angular/cdk/bundles/cdk-scrolling.umd.js',
+    '@angular/cdk/table':'npm:/@angular/cdk/bundles/cdk-table.umd.js'
```
Comment (DevVersion) [inline guides/getting-started.md]: At first glance, it looks like the `stepper` is missing here.

Comment (donroyco) [reply guides/getting-started.md]: That's true, @shaunak1111 see here for the list of `cdk` bundles: https://github.com/angular/cdk-builds/tree/master/bundles

Comment (shaunak1111) [reply guides/getting-started.md]: @DevVersion @donroyco I have added the necessary cdk modules stepper and layout in the docs and raised a PR.

https://github.com/angular/material2/pull/7445

**Comment by josephperrott** [comment]
@DevVersion Any thoughts on automatically updating this portion of `getting-starting.md` automatically?

I worry about it falling out of date, but having language in the description that implies its a complete list.

**Comment by jelbourn** [comment]
Closing this since it's become quite stale. Feel free to open a new PR if this is still relevant.

**Comment by Synoon** [comment]
I've had to add all these packages to systemjs to get material runable  (Version 5.0.1)

`                          //Angular Material
                          '@angular/material': 'node_modules/@angular/material/bundles/material.umd.js',
                          '@angular/animations': 'node_modules/@angular/animations/bundles/animations.umd.js',
                          '@angular/animations/browser': 'node_modules/@angular/animations/bundles/animations-browser.umd.js',
                          '@angular/platform-browser/animations': 'node_modules/@angular/platform-browser/bundles/platform-browser-animations.umd.js',
                          //Angular Material - CDK individual packages 
                          '@angular/cdk': 'node_modules/@angular/cdk/bundles/cdk.umd.js',
                          '@angular/cdk/a11y': 'node_modules/@angular/cdk/bundles/cdk-a11y.umd.js',
                          '@angular/cdk/bidi': 'node_modules/@angular/cdk/bundles/cdk-bidi.umd.js',
                          '@angular/cdk/coercion': 'node_modules/@angular/cdk/bundles/cdk-coercion.umd.js',
                          '@angular/cdk/collections': 'node_modules/@angular/cdk/bundles/cdk-collections.umd.js',
                          '@angular/cdk/keycodes': 'node_modules/@angular/cdk/bundles/cdk-keycodes.umd.js',
                          '@angular/cdk/layout': 'node_modules/@angular/cdk/bundles/cdk-layout.umd.js',
                          '@angular/cdk/observers': 'node_modules/@angular/cdk/bundles/cdk-observers.umd.js',
                          '@angular/cdk/overlay': 'node_modules/@angular/cdk/bundles/cdk-overlay.umd.js',
                          '@angular/cdk/platform': 'node_modules/@angular/cdk/bundles/cdk-platform.umd.js',
                          '@angular/cdk/portal': 'node_modules/@angular/cdk/bundles/cdk-portal.umd.js',
                          '@angular/cdk/rxjs': 'node_modules/@angular/cdk/bundles/cdk-rxjs.umd.js',
                          '@angular/cdk/scrolling': 'node_modules/@angular/cdk/bundles/cdk-scrolling.umd.js',
                          '@angular/cdk/stepper': 'node_modules/@angular/cdk/bundles/cdk-stepper.umd.js',
                          '@angular/cdk/table': 'node_modules/@angular/cdk/bundles/cdk-table.umd.js',
                          '@angular/cdk/accordion': 'node_modules/@angular/cdk/bundles/cdk-accordion.umd.js',
                          '@angular/common/http': 'node_modules/@angular/common/bundles/common-http.umd.js',
                          'tslib': 'node_modules/tslib/tslib',`

**Pull request closed without merging**